### DISCUSSION TITLE
Changing Languages Widget to Include More Languages per Row

### DISCUSSION THREAD

**Discussion by VinayR-GitHub** [anuraghazra/github-readme-stats#3557] [Q&A] [answered]
I am new to JavaScript and have tried on a fork to change the number of columns of languages present. Is there a change I could make to the original card so that there are '4' languages per row instead of '2', and they increase from left-to-right instead of top-to-bottom?

**Comment by rickstaa** [comment, marked as answer]
@VinayR-GitHub, thanks for asking! Currently, we support modifying displayed languages only through the `langs_count` parameter. For more details, visit [Language Card Options](https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#language-card-exclusive-options). If you want to have your feature implemented, you can create [a pull request](https://github.com/anuraghazra/github-readme-stats/compare) or [Feature Request](https://github.com/anuraghazra/github-readme-stats/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.yml). If there is enough interest from the community, we might take a look (see [Issue #1935](https://github.com/anuraghazra/github-readme-stats/issues/1935)) 👍🏻.
